css/orderpage.css

```diff
@@ -0,0 +1,191 @@
+
+ :root {
+    --grey-dark: #848993;
+    --grey-darker: #4D5057;
+    --grey-light: #E5E5E5;
+    --orange-dark: #E0633A;
+    --orange-light: #FCEFEB;
+    --white: #fff;
+    --white-transparent: rgba(255,255,255,0.9); 
```

Comment:
If the layout for the form and image is corrected on the small screen, this is not needed